modules/ROOT/pages/6.7-release-notes.adoc

```diff
@@ -335,6 +335,11 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 === Links in **Help → Help → Plugins** and **Help → Help → Version** were not navigable by keyboard
 // TINY-10071
+Accessibility of links within the Plugins Tab of the Help Dialog could not be navigable to the user as these links lacked the essential attribute, `data-alloy-tabstop="true"` rendering them inaccessible via keyboard navigation.
+
+Consequently, users were unable to navigate through these links using the tab key.
+
+{productname} 6.7, implemented a fix which involved adding the missing `data-alloy-tabstop="true"` attribute to these links. As a result of this, users can now efficiently navigate through the links within the Plugins Tab of the Help Dialog using their keyboards.
```

Comment:
```suggestion
The Help dialog built-in to {productname} includes four tabs:
* *Handy Shortcuts*;
* *Keyboard Navigation*;
* *Plugins*; and
* *Version*.

Previously, keyboard navigation was supported across these four tabs, and into each tab’s contents. (When a tab’s contents have focus, the content can be scrolled through using Arrow keys.)

However, keyboard-based navigation of links in the focussed content was not supported.

{productname} 6.7 corrects this. The `data-alloy-tabstop="true"` attribute, which was previously not applied to these links, has been added.

Users of {productname} 6.7 can navigate to and activate links in Help dialog contents entirely with keyboard navigation.

[NOTE]
====
As of this release, only the *Plugins*, and *Version* tabs include links.

Also, there is no visual feedback when a given tab’s contents pane takes focus.

Consequently, the UX is such that it appears to take two presses of the *Tab* key to go from a tab label having the focus and the first link within a tab’s content pane having focus.

Likewise, going from the first link within a tab’s content pane having focus to a tab label having focus appears to take two presses of the *Shift-Tab* keyboard chord.
====
```
I believe calling out the absence of visual feedback for the tab’s contents pane taking focus is worthwhile.

Reply:
I agree @brian-forte , do you think its worth adding that when the link takes `focus` it now renders a`focus` border also?

Reply:
We’ll document that in the entry for that ticket in the Improvements section of the release notes:
```
=== Improved the keyboard focus visibility for links inside dialogs
// TINY-10124
```